### PR TITLE
More detail on claim campaigns rewards' notification

### PR DESCRIPTION
**Pull request by nguyenhuudungz**

### CODE REVIEW

File: src/pages/Campaign/CampaignButtonWithOptions.tsx [hunk 1]
```diff
@@ -83,11 +83,27 @@ export default function CampaignButtonWithOptions({
       const encodedData = response.data.data.EncodedData
       try {
         await sendTransaction(rewardContractAddress, encodedData, BigNumber.from(0), transactionResponse => {
-          // TODO nguyenhuudungz: Compile a list of unclaimed rewards from `campaignLeaderboard`.
+          const accumulatedUnclaimedRewards = selectedCampaignLeaderboard.rewards
+            .filter(reward => !reward.claimed)
```
Comment (nguyenhoaidanh) [inline src/pages/Campaign/CampaignButtonWithOptions.tsx:87]: we can combine **filter and reduce** => only reduce

Comment (namgold) [reply src/pages/Campaign/CampaignButtonWithOptions.tsx:87]: I prefer make it separated. Just my prefered.

Comment (nguyenhoaidanh) [reply src/pages/Campaign/CampaignButtonWithOptions.tsx:87]: if **selectedCampaignLeaderboard** is not large, it is ok.

File: src/pages/Campaign/CampaignButtonWithOptions.tsx [hunk 2]
```diff
@@ -83,11 +83,27 @@ export default function CampaignButtonWithOptions({
...
+                acc[key] = {
+                  ...value,
+                  rewardAmount: value.rewardAmount.add(acc[key].rewardAmount),
+                }
+              }
+              return acc
+            }, {})
+          const rewardString = Object.values(accumulatedUnclaimedRewards)
+            .map(reward => reward.rewardAmount.toSignificant(DEFAULT_SIGNIFICANT) + ' ' + reward.token.symbol)
+            .join(' ' + t`and` + ' ')
```
Comment (namgold) [inline src/pages/Campaign/CampaignButtonWithOptions.tsx:102]: Give [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/ListFormat) a try if it is better

**Review by nguyenhoaidanh — APPROVED** [review]

File: src/pages/Campaign/CampaignButtonWithOptions.tsx (hunk 1 above)
Comment (nguyenhuudungz) [reply src/pages/Campaign/CampaignButtonWithOptions.tsx:87]: yeah it's usually 1 or 2 items

Comment (nguyenhoaidanh) [reply src/pages/Campaign/CampaignButtonWithOptions.tsx:87]: okay.

File: src/pages/Campaign/CampaignButtonWithOptions.tsx (hunk 2 above)
Comment (nguyenhuudungz) [reply src/pages/Campaign/CampaignButtonWithOptions.tsx:102]: interesting bro but hard to read, not natural i think

**Pull request merged**